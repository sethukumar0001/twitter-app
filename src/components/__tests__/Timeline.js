import React from 'react';
import renderer from 'react-test-renderer';
import Timeline from '../Timeline';

describe('Timeline', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <Timeline>
        <li>tweet</li>
      </Timeline>,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
