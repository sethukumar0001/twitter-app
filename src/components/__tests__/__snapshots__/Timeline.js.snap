// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timeline renders correctly 1`] = `
<ul
  className="Timeline-timeline-1"
>
  <li>
    tweet
  </li>
</ul>
`;
